### PR TITLE
[Log Analytics] Fix #25660: az monitor log-analytics workspace table update --total-retention-time fix upper validation.

### LINKED ISSUE #25660: az monitor log-analytics workspace table update validation wrong
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Related command**

`az monitor log-analytics workspace table update --name {Table} --retention-time 730 --total-retention-time 2556 --resource-group {Resource Group} --workspace-name {Workspace}`

**Describe the bug**

Total Retention time should be able to be set to 2556, but azcli does not validate this value. 
Az CLI allows 2555, but API reponds with error:
"The specified total retention value is invalid. The value beyond two years is restricted to full years. Allowed values are: [4-730], 1095,1460, 1826, 2191, 2556 days"

**To Reproduce**

Run command:
`az monitor log-analytics workspace table update --name {Table} --retention-time 730 --total-retention-time 2556 --resource-group {Resource Group} --workspace-name {Workspace}`
Errors with Validation.

`az monitor log-analytics workspace table update --name {Table} --retention-time 730 --total-retention-time 2555 --resource-group {Resource Group} --workspace-name {Workspace}`
Errors with API response.

**Expected behavior**

Value should update as documented.

**Environment summary**

**Additional context**

### PR DESCRIPTION
**Pull request by ryan-royals**
**Related command**

 az monitor log-analytics workspace table update --name {TABLE} --retention-time 730 --total-retention-time 2556 --resource-group {RESOURCEGROUP} --workspace-name {WORKSPACE}

**Description**

Previously command would fail with total retention time of `2556` as it was only allowing a value of up to 2555. The ARM API only accepts [4-730], 1095, 1460, 1826, 2191, 2556  

**Testing Guide**

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
monitor

File: src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/workspace/table/_update.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
             arg_group="Properties",
             help="The table total retention in days, between 4 and 2555.",
             fmt=AAZIntArgFormat(
-                maximum=2555,
+                maximum=2556,
                 minimum=4,
```
Comment (AllyW) [inline src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/workspace/table/_update.py:105]: Auto generated code needs to be updated in AAZ data model.

Comment (AllyW) [reply src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/workspace/table/_update.py:105]: Please refer to https://github.com/Azure/azure-rest-api-specs/blob/main/specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2021-12-01-preview/Tables.json#L656

Comment (ryan-royals) [reply src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/workspace/table/_update.py:105]: @AllyW is this a task you are asking of me? I don't understand what the next step is to getting my fix merged.

Comment (AllyW) [reply src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/workspace/table/_update.py:105]: Hi @ryan-royals , thanks for code testing and fixing. The thing is, cli code is auto-generated by azure service docs posted upside, so this problem should be solved by service doc update and I'm following it. Therefore, code change here is not enough and I'll keep you updated if anything changes.

Comment (ryan-royals) [reply src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/workspace/table/_update.py:105]: Hi @AllyW, any updates on this?

Comment (AllyW) [reply src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/workspace/table/_update.py:105]: hi @ryan-royals, service does not schedule this doc change recently and we are planning to apply your changes in this sprint. As is known to you that, _update.py is generated code by doc. If we need to adjust the `min` and `max` validators, the correct way is inheriting from this generated cmd and editing its fmt in new class. I'm going to do this in this sprint and link the pr here. Also if you're interested, you can refer to this pr https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/network/operations/watcher.py#L333

**Pull request closed without merging**